src/style.css

```diff
@@ -1,3 +1,133 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  height: 60px;
+  box-sizing: border-box;
+}
+
+.nav-logo a {
+  display: flex;
+  align-items: center;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: flex-end;
+}
+
+ul {
```

Comment:
Avoid using tag names for styling. You should use classes instead of tag names like 'ul' to apply styles, except for the 'html' and 'body' tags.